BUILDING.md

```diff
@@ -1,29 +1,28 @@
 # Getting Started with Create React App
 
-This project is build with [Vite](https://vitejs.dev/)
+This project is built with [Vite](https://vitejs.dev/)
 
 ## Available Scripts
 
 In the project directory, you can run:
 
 ### `yarn dev`
 
-Runs the app in the development mode.\
+This command runs the app in development mode.\
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+The page reloads when you make edits.\
+Lint errors also appear in the console.
 
-### `yarn build`
-
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### `yarn preview`
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+This command runs the app in a local preview production build.
 
-See the section about [Building for Production](https://vitejs.dev/guide/build.html) for more information.
+### `yarn build`
```

Comment:
I think it would be best to just change it to something like

```
This command creates an optimized production build and places it in the `build` folder.

See this section about [Building for Production](https://vitejs.dev/guide/build.html) for more information.
```

Reply:
last thing, `command optimizes the production build` makes it sound like it's optimizing an existing build, but it creates the optimized production build

Reply:
Missed the notification for this. Please let me know if the change I just made is sufficient.

Also, is there a better way to contact you? Discord maybe?

Reply:
you can join the server https://discord.gg/DDZdqZWaHA